lib/random.py

```diff
@@ -0,0 +1,28 @@
+"""Generate pseudo random numbers. Should not be used for security purposes."""
+
+from __go__.math.rand import Float64, Seed
+from __go__.time import Now
+
+
+# default seed with time like in cpython
+Seed(Now().UnixNano())
```

Comment:
Because Grumpy modules may be used by plain Go programs, I'm inclined to just rely on Go's default seeding behavior. This will result in a divergence in the way CPython and Grumpy programs execute, but issues that arise from the difference would be problems with the program, not with Grumpy.